addons/base_address_extended/models/res_company.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = 'res.company'
+
+    street_name = fields.Char('Street Name', compute='_compute_address',
+                              inverse='_inverse_street')
+    street_number = fields.Char('House Number', compute='_compute_address',
+                                inverse='_inverse_street')
+    street_number2 = fields.Char('Door Number', compute='_compute_address',
+                                 inverse='_inverse_street')
+
+    def _get_company_address_fields(self, partner):
+        address_fields = super(Company, self)._get_company_address_fields(partner)
+        address_fields.update({
+            'street_name': partner.street_name,
+            'street_number': partner.street_number,
+            'street_number2': partner.street_number2,
+        })
+        return address_fields
+
+    def _inverse_street(self):
+        for company in self:
```

Comment:
The `_inverse_street` in the base module also writes on the `partner_id.street` field.
We don't call super and we don't write that field anymore, I guess that's intended?

I know you just moved the code, but it seems weird.